.github/workflows/rhel-9.yml

```diff
@@ -763,3 +763,112 @@ jobs:
           path: |
             /tmp/artifacts/*.json
             /tmp/artifacts/*.log
+
+  pre-edge-vsphere-9:
+    needs: pr-info
+    runs-on: ubuntu-latest
+    env:
+      STATUS_NAME: Tests / edge-vsphere-9
+
+    steps:
+      - name: Get Current Job Log URL
+        id: job-log-url
+        run: |
+          JOB_HTML_URL=$(curl --get -Ss -H "Authorization: token ${GITHUB_TOKEN}" -H "Accept: application/vnd.github.v3+json" "https://api.github.com/repos/osbuild/rhel-edge-ci/actions/runs/${GITHUB_RUN_ID}/jobs?per_page=30" | jq -r --arg job_name "pre-edge-vsphere-9" '.jobs | map(select(.name == "pre-edge-vsphere-9")) | .[0].html_url')
+          echo "html_url=$JOB_HTML_URL" >> $GITHUB_OUTPUT
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Create in-progress status
+        uses: octokit/request-action@v2.x
+        with:
+          route: 'POST /repos/osbuild/osbuild-composer/statuses/${{ needs.pr-info.outputs.sha }}'
+          context: ${{ env.STATUS_NAME }}
+          state: pending
+          description: 'RHEL 9.3 - Runner has been deploying...'
+          target_url: '${{ steps.job-log-url.outputs.html_url }}'
+        env:
+          GITHUB_TOKEN: ${{ secrets.PAT }}
+
+  edge-vsphere-9:
+    needs: [pr-info, pre-edge-vsphere-9]
+    runs-on: [kite, x86_64, rhos-0x, rhel-9-3, large]
```

Comment:
Please change the instance type to medium

Reply:
done.